usaspending_api/etl/submission_loader_helpers/disaster_emergency_fund_codes.py

```diff
@@ -11,11 +11,12 @@ def get_disaster_emergency_fund(row: dict) -> Optional[dict]:
     global DISASTER_EMERGENCY_FUND_CODES
     if not DISASTER_EMERGENCY_FUND_CODES:  # testing for falsy values instead of just null
         DISASTER_EMERGENCY_FUND_CODES = {defc.code: defc for defc in DisasterEmergencyFundCode.objects.all()}
+
     disaster_emergency_fund_code = row["disaster_emergency_fund_code"]
     if not disaster_emergency_fund_code:
         return None
     try:
-        return DISASTER_EMERGENCY_FUND_CODES[disaster_emergency_fund_code]
+        return DISASTER_EMERGENCY_FUND_CODES[disaster_emergency_fund_code.upper()]
```

Comment:
Stepping through the code more to re-familiarize myself I don't believe you need to make a change here. This method is only used when loading File C. Prior to calling this method a helper method is used to handle uppercasing all strings found in each File C row. See here https://github.com/fedspendingtransparency/usaspending-api/blob/fecd5ec8085ccd306cdadde85d251219b16a5914/usaspending_api/etl/submission_loader_helpers/file_c.py#L164.